test/compilable/extra-files/ddoc17697.html

```diff
@@ -1,484 +1,506 @@
-
```

Comment:
Wait I just copied the produced file over the other, so technically if I put them in different commits I create artificial steps that don't exist. So what difference does this line make if it's in a separate commit?